### PR TITLE
Rename marriage-abroad services

### PR DESCRIPTION
**Pull request by chrisroos**
Trello card: https://trello.com/c/qMeYVdfh

This standardises the names of marriage-abroad services based on the information in the spreadsheet attached to the Trello card.

I'm hoping to get this merged and then update @erikse's PR #2404 to extract rates from the template into a data file.

## Expected changes

These changes affect a lot of outcomes. The following list contains examples of each of the changes introduced in the commits in this PR:

### Austria

* [Before](https://www.gov.uk/marriage-abroad/y/austria/ceremony_country/partner_british/same_sex#fees) and [after](https://smart-answers-pr-2413.herokuapp.com/marriage-abroad/y/austria/ceremony_country/partner_british/same_sex#fees)
* Replace: "Issuing a CNI or equivalent" with "Issuing a CNI, Nulla Osta or equivalent"

### Albania

* [Before](https://www.gov.uk/marriage-abroad/y/albania/ceremony_country/partner_british/opposite_sex#fees) and [after](https://smart-answers-pr-2413.herokuapp.com/marriage-abroad/y/albania/ceremony_country/partner_british/opposite_sex#fees)
* Remove: "Issuing any other consular letter or certificate in English"
* Remove: "Issuing any other consular letter or certificate in any other language"
* Remove: "Administering an oath or making a declaration"

### Cambodia

* [Before](https://www.gov.uk/marriage-abroad/y/cambodia/ceremony_country/partner_british/same_sex#fees) and [after](https://smart-answers-pr-2413.herokuapp.com/marriage-abroad/y/cambodia/ceremony_country/partner_british/same_sex#fees).
* Reword: "Registering a marriage or civil partnership" to "Registering a civil partnership or marriage"
* Reword: "Issuing a marriage or civil partnership certificate" to "Issuing a civil partnership or marriage certificate"

### Colombia

* [Before](https://www.gov.uk/marriage-abroad/y/colombia/ceremony_country/partner_british/same_sex#fees) and [after](https://smart-answers-pr-2413.herokuapp.com/marriage-abroad/y/colombia/ceremony_country/partner_british/same_sex#fees).
* Replace: "Posting a marriage notice" with "Receiving a notice of registration"
* Replace: "Performing a marriage ceremony" with "Registering a marriage"

### Qatar

* [Before](https://www.gov.uk/marriage-abroad/y/qatar/ceremony_country/partner_other/opposite_sex#fees) and [after](https://smart-answers-pr-2413.herokuapp.com/marriage-abroad/y/qatar/ceremony_country/partner_other/opposite_sex#fees).
* Replace: "Affidavit for marriage" with "Affidavit or affirmation for marriage"
* Remove: "Issuing any other consular letter or certificate in English"
* Remove: "Issuing any other consular letter or certificate in any other language"

### Finland

* [Before](https://www.gov.uk/marriage-abroad/y/finland/third_country/partner_local/opposite_sex#fees) and [after](https://smart-answers-pr-2413.herokuapp.com/marriage-abroad/y/finland/third_country/partner_local/opposite_sex#fees).
* Replace: "Affirmation for marriage" with "Affidavit or affirmation for marriage"

### Lebanon

* [Before](https://www.gov.uk/marriage-abroad/y/lebanon/uk/partner_local/opposite_sex#fees) and [after](https://smart-answers-pr-2413.herokuapp.com/marriage-abroad/y/lebanon/uk/partner_local/opposite_sex#fees).
* Replace: "Affirmation for freedom to marry in English" with "Affidavit or affirmation for marriage"
* Remove: "Affirmation for freedom to marry in any other language"

### China

* [Before](https://www.gov.uk/marriage-abroad/y/china/uk/partner_other/opposite_sex#fees) and [after](https://smart-answers-pr-2413.herokuapp.com/marriage-abroad/y/china/uk/partner_other/opposite_sex#fees).
* Replace: "Affirmation or affidavit for freedom to marry in English & Chinese" with "Affidavit or affirmation for marriage"

### Peru

* [Before](https://www.gov.uk/marriage-abroad/y/peru/uk/partner_british/opposite_sex#fees) and [after](https://smart-answers-pr-2413.herokuapp.com/marriage-abroad/y/peru/uk/partner_british/opposite_sex#fees).
* Replace: "Affirmation or affidavit for freedom to marry" with "Affidavit or affirmation for marriage"

### Egypt

* [Before](https://www.gov.uk/marriage-abroad/y/egypt/uk/partner_local/opposite_sex#fees) and [after](https://smart-answers-pr-2413.herokuapp.com/marriage-abroad/y/egypt/uk/partner_local/opposite_sex#fees).
* Replace: "Marriage declaration" with "Affidavit or affirmation for marriage"
* Remove: "Making an oath"

### CODE REVIEW

File: lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_japan.govspeak.erb [hunk 1]
```diff
@@ -50,7 +50,6 @@
   <%= render partial: 'consular_fees_table.govspeak.erb',
              locals: {
                services: [
-                 :administering_oath_or_making_declaration,
```
Comment (ikennaokpala) [inline lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_japan.govspeak.erb]: Should we still keep this partial here.. it appears empty.. isn't it best to leave this out.. see screenshots

## GOVUK
screen shot 2016-03-30 at 14 16 30


## My Localhost running this branch


screen shot 2016-03-30 at 14 17 35

File: lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_belarus.govspeak.erb [hunk 2]
```diff
@@ -41,7 +41,6 @@
   <%= render partial: 'consular_fees_table.govspeak.erb',
              locals: {
                services: [
-                 :administering_oath_or_making_declaration,
```
Comment (ikennaokpala) [inline lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_belarus.govspeak.erb]: This as well appears empty.. to the users.. please see my previous screenshot.

```erb
<% if services.present? %>
  ## Fees

  Service | Fee
  -|-
  <%= render partial: 'consular_fees_table_items.govspeak.erb', collection: services, as: :service %>
<% end %>
```

File: lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_oman.govspeak.erb [hunk 3]
```diff
@@ -41,7 +41,6 @@
   <%= render partial: 'consular_fees_table.govspeak.erb',
              locals: {
                services: [
-                 :administering_oath_or_making_declaration,
```
Comment (ikennaokpala) [inline lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_oman.govspeak.erb]: See comments and screenshots [here](https://github.com/alphagov/smart-answers/pull/2413/files#r57885971)

File: test/artefacts/marriage-abroad/belarus/ceremony_country/partner_local/opposite_sex.txt [hunk 4]
```diff
@@ -40,7 +40,6 @@ You must take the form with you to your appointment.
 
 Service | Fee
 -|-
-Administering an oath or making a declaration | £55
```
Comment (ikennaokpala) [inline test/artefacts/marriage-abroad/belarus/ceremony_country/partner_local/opposite_sex.txt]: See comments and screenshots [here](https://github.com/alphagov/smart-answers/pull/2413/files#r57885971)

**Comment by ikennaokpala** [comment]
Hi @chrisroos great piece of work.. I found your very preview links useful.. I observed that the fees tables for Belarus, Japan and Oman appear empty.. is this intentional ??

**Comment by chrisroos** [comment]
Thanks for reviewing this, @ikennaokpala. And thanks for picking up on the empty fees tables! I'll investigate those and work out what we should do about them.

**Comment by chrisroos** [comment]
I'm currently working on an alternative to this branch. I'm extracting all the service/fee data from the templates into a yaml file. I think this is going to make it easier to both rename the services, and update the services offered by the various countries (both required by 6 April). I think it's also going to make the templates simpler.

I'm going to close this PR and will have the new one open by Tuesday.

**Comment by chrisroos** [comment]
The alternative approach to updating these services is in PR #2438.

**Pull request closed without merging**